fog/view/server/src/bin/router.rs

```diff
@@ -34,17 +37,24 @@ fn main() {
     );
 
     // TODO: Remove and get from a config.
-    let mut shard_uris: Vec<FogViewStoreUri> = Vec::new();
+    let mut fog_view_grpc_clients= Vec::new();
+    let grpc_env = Arc::new(
+        grpcio::EnvBuilder::new()
+            .name_prefix("Main-RPC".to_string())
+            .build(),
+    );
     for i in 0..50 {
-        let shard_uri_string = format!(
-            "insecure-fog-view-store://node{}.test.mobilecoin.com:3225",
-            i
+        let shard_uri_string = format!("insecure-fog-view://node{}.test.mobilecoin.com:3225", i);
```

Comment:
I think that makes sense. It's a bit of a larger change because now we have to have the FogViewService take either a FogViewStoreUrl OR FogViewUri. It will probably require some sort of trait to make this generic? Anyway, I'd like to implement this in a follow-up PR.

Reply:
This makes sense. It's a bit of a larger change to incorporate the two URI schemes since it necessitates modifying the `FogViewService` to take two different concrete URI implementations. It's not that much work but it is sorta orthogonal to this PR so I'd like to implement this change in a follow up PR if that's cool.

Reply:
This makes sense. It's a bit of a larger change to incorporate the two URI schemes since it necessitates modifying the FogViewService to take two different concrete URI implementations. It's not that much work but it is sorta orthogonal to this PR so I'd like to implement this change in a follow up PR if that's cool.

Reply:
This makes sense. It's a bit of a larger change to incorporate the two URI schemes since it necessitates modifying the FogViewService to take two different concrete URI implementations. It's not that much work but it is sorta orthogonal to this PR so I'd like to implement this change in a follow up PR if that's cool.